docs/courses/robotics/3.2-serial-communication.md

```diff
@@ -10,6 +10,27 @@ Design and execute software to read data from a mass balance using serial commun
 
 ## 🔰 Tutorial
 
+Serial communication is a method of transmitting data between devices one bit at a time. It's widely used in scientific instruments, industrial equipment, and embedded systems due to its simplicity and reliability. In this tutorial, we'll focus on RS232, a common serial communication standard, and how to interface it with modern microcontrollers.
```

Comment:
Maybe briefly clarify what "serial" means in the context of serial communication